src/main/java/core/basesyntax/FigureBase.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+abstract class FigureBase implements FigureArea, FigureDrawable {
+    protected String color;
+
+    public FigureBase(String color) {
+        this.color = color;
+    }
+
+    @Override
+    public abstract double getArea();
+
+    @Override
+    public abstract void getInfo();
```

Comment:
you can skip these declarations as they're already declared in corresponding interfaces